org.nongnu.gsequencer.gsequencer.json

```diff
@@ -154,24 +158,79 @@
+            ],
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=RelWithDebInfo",
+                "-DSMTG_ADD_VST3_PLUGINS_SAMPLES=OFF",
+                "-DSMTG_ADD_VST3_HOSTING_SAMPLES=OFF"
+            ],
+            "post-install": [
+                "install -d ${FLATPAK_DEST}/include/vst3",
+                "cp -rv ../{base,pluginterfaces,public.sdk,vstgui4} ${FLATPAK_DEST}/include/vst3",
+                "install -d ${FLATPAK_DEST}/lib/vst3",
+                "cp -v lib/RelWithDebInfo/*.a ${FLATPAK_DEST}/lib/vst3"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/steinbergmedia/vst3sdk.git",
+                    "commit": "138ac4eb7332d44873fe8ca2e326887ab8ef671a"
+                },
+                {
+                    "type": "patch",
+                    "path": "patches/vst3sdk.patch"
+                }
+            ]
+        },
         {
             "name": "gsequencer",
+            "rm-configure": true,
             "config-opts": [
                 "--disable-introspection",
-		"HTMLHELP_XSL=/usr/share/xml/docbook/xml/xsl-stylesheets/htmlhelp/htmlhelp.xsl"
+                "--enable-vst3",
+                "HTMLHELP_XSL=/usr/share/xml/docbook/xml/xsl-stylesheets/htmlhelp/htmlhelp.xsl"
             ],
+            "build-options": {
+                "env": {
+		    "VST3_RELEASE_TYPE": "-DRELEASE",
+                    "VST3_CXXFLAGS": "-DRELEASE -I/app/include/vst3",
+                    "VST3_LIBS": "-L/app/lib/vst3"
+                }
+            },
             "post-install": [
                 "make html",
                 "make install-html-mkdir",
                 "make install-html",
                 "sed -i -e 's~Icon=/app/share/gsequencer/icons/ags.png~Icon=org.nongnu.gsequencer.gsequencer.png~g' /app/share/applications/gsequencer.desktop",
+                "install -Dm755 COPYING -t /app/share/gsequencer/GPL-3",
                 "install -d /app/extensions/Plugins"
             ],
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://download-mirror.savannah.gnu.org/releases/gsequencer/3.9.x/gsequencer-3.9.3.tar.gz",
-                    "sha256": "dea1231e66d4e4fe1327cead03489f0431e372cf029035e2ea3b797713aadc06"
+                    "url": "https://download-mirror.savannah.gnu.org/releases/gsequencer/3.10.x/gsequencer-3.10.8.tar.gz",
+                    "sha256": "b1ea3cde943bbc46e4ddb8635b21be66458995e87014bae4026137e146cc78cd"
+                },
+                {
+                    "type": "patch",
+                    "path": "patches/gsequencer.patch"
+                },
+                {
+                    "type": "script",
+                    "dest-filename": "autogen.sh",
+                    "commands": [
+                        "autoreconf -vif"
+                    ]
                 },
                 {
```

Comment:
the change in the post-install was meant to make this `source` unecessary. (it was illogical)

Reply:
configure failed

Reply:
GPL-3 is showed within UI

Reply:
it is installed in the `post-install`.
This is for steps before the build.